### DISCUSSION TITLE
Rain wallpaper video support.

### DISCUSSION THREAD

**Discussion by ghost** [rocksdanister/lively#783] [Ideas]
Hello, I like the Rain wallpaper very much but I think it would be even better if you could set a video/gif as a background instead of just color or a still image. Shouldn't be too hard to implement. :)

**Comment by neoscotch** [comment]
it already does video according to the docs https://github.com/rocksdanister/lively#features. I verified myself that drag and drop for videos works just fine on windows 11

**Reply by rocksdanister** [reply]
No, what is being asking for is video texture support in default Rain v2 wallpaper to have rain particles on video files.

**Reply by neoscotch** [reply]
ok I removed my thumbs down. I guess I didn't understand the context. thanks!

**Comment by erwinwallpaper** [comment]
Nice idea
